nix/rust-builder.nix

```diff
@@ -48,10 +48,15 @@ let
 
   rustToolchain =
     if useRustNightly
-    then pkgs.rust-bin.selectLatestNightlyWith (toolchain: toolchain.default)
+    then pkgs.rust-bin.selectLatestNightlyWith (toolchain: toolchain.default.override {
+          extensions = [ "llvm-tools-preview" ];
+        })
```

Comment:
Nightly only builds documentation and tests as a second option, no need to do code coverage there.

Reply:
Ok